src/transformers/models/layoutlm/tokenization_layoutlm.py

```diff
@@ -34,30 +38,491 @@
     }
 }
 
-
 PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES = {
     "microsoft/layoutlm-base-uncased": 512,
     "microsoft/layoutlm-large-uncased": 512,
 }
 
-
 PRETRAINED_INIT_CONFIGURATION = {
     "microsoft/layoutlm-base-uncased": {"do_lower_case": True},
     "microsoft/layoutlm-large-uncased": {"do_lower_case": True},
 }
 
 
-class LayoutLMTokenizer(BertTokenizer):
+# Copied from transformers.models.bert.tokenization_bert.load_vocab
+def load_vocab(vocab_file):
+    """Loads a vocabulary file into a dictionary."""
+    vocab = collections.OrderedDict()
+    with open(vocab_file, "r", encoding="utf-8") as reader:
+        tokens = reader.readlines()
+    for index, token in enumerate(tokens):
+        token = token.rstrip("\n")
+        vocab[token] = index
+    return vocab
+
+
+# Copied from transformers.models.bert.tokenization_bert.whitespace_tokenize
+def whitespace_tokenize(text):
+    """Runs basic whitespace cleaning and splitting on a piece of text."""
+    text = text.strip()
+    if not text:
+        return []
+    tokens = text.split()
+    return tokens
+
+
+class LayoutLMTokenizer(PreTrainedTokenizer):
```

Comment:
I think you can apply the Copied from on the entire class without any issue, `VOCAB_FILES_NAMES` and all other constants will use the value defined in this module.

Reply:
Oh yeah, didn't realize that! Added Copied statement to the class :)